lib/justifi/dispute.rb

```diff
@@ -3,8 +3,13 @@
 module Justifi
   module Dispute
     class << self
-      def list(params: {}, headers: {}, seller_account_id: nil)
-        headers[:seller_account] = seller_account_id if seller_account_id
+      def list(params: {}, headers: {}, seller_account_id: nil, sub_account_id: nil)
+        if seller_account_id
+          Justifi.seller_account_deprecation_warn
+          headers[:seller_account] = seller_account_id
+        end
+        headers[:sub_account] = sub_account_id if sub_account_id
```

Comment:
I think we can change this to something like, same for the others too, WDYT?
```suggestion
        Justifi.seller_account_deprecation_warn if seller_account_id
        
        headers[:sub_account_id] = sub_account_id || seller_account_id
```

Reply:
I like it; just one thing, we will not see a request with the seller-account header even if the user of the gem uses the seller_account_id parameter.

Reply:
Besides that, I think it's a good call. Thanks!